src/Bundler.js

```diff
@@ -458,9 +458,9 @@ class Bundler extends EventEmitter {
     return Server.middleware(this);
   }
 
-  async serve(port = 1234) {
-    this.bundle();
-    return await Server.serve(this, port);
+  async serve(port = 1234, host) {
+    await this.bundle();
+    return await Server.serve(this, port, host);
```

Comment:
This was resulting in output like this:

![image](https://user-images.githubusercontent.com/4730164/34023570-a0d27dd6-e113-11e7-83dc-e0a26e35d426.png)

Instead of like this:

![image](https://user-images.githubusercontent.com/4730164/34023581-b05fa486-e113-11e7-97cc-8ac0f611dea6.png)

Seems the `persistent` server logging should come at the end, and not be shown until everything is ready.

Reply:
Caused by #164, build output was on a single line prior to that. The _build_ time will remain the same with this change, but there could be a minor delay added between end of build and dev server availability due to the blocking. Not a big deal IMO, it should be on the magnitude of milliseconds.

Alternatives:
- Revert this, leave the broken up logging
- Refactor logging of server messages until build is finished

Reply:
... whereas here you are setting the default port but not the default host. Can't the defaults be in the same place?

Reply:
@rondonjon - good call! Updated to handle defaults with `commander`. This also gives us nicer help commands.